armbian/base/scripts/bbb-config.sh

```diff
@@ -0,0 +1,210 @@
+#!/bin/bash
+set -e
```

Comment:
Maybe we should also set `-u`, to catch undefined variables being used?

(And elsewhere.)